packages/x-license/src/verifyLicense/verifyLicense.ts

```diff
@@ -178,15 +198,17 @@ export function verifyLicense({
     return { status: LICENSE_STATUS.Invalid };
   }
 
-  if (license.planVersion === 'initial') {
-    // 'charts-pro' or 'tree-view-pro' can only be used with a newer license
-    if (productScope === 'charts' || productScope === 'tree-view') {
-      return { status: LICENSE_STATUS.NotAvailableInInitialProPlan };
-    }
+  if (!isLicenseScopeSufficient(packageName, license.scope)) {
+    return { status: LICENSE_STATUS.OutOfScope };
   }
 
-  if (!acceptedScopes.includes(license.scope)) {
-    return { status: LICENSE_STATUS.OutOfScope };
+  // 'charts-pro' or 'tree-view-pro' can only be used with a newer Pro license
```

Comment:
The order change is here to make sure that a pro license used on a premium package always have the `OutOfScope` status and not the `NotAvailableInInitialProPlan`